src/nxt_main_process.c

```diff
@@ -323,13 +323,78 @@ static nxt_conf_map_t  nxt_java_app_conf[] = {
 };
 
 
+static nxt_conf_map_t  nxt_wasm_app_conf[] = {
+    {
+        nxt_string("module"),
+        NXT_CONF_MAP_CSTRZ,
+        offsetof(nxt_common_app_conf_t, u.wasm.module),
+    },
+
+    {
```

Comment:
While some structures above use these blanks, the ones right above don't.  I guess you'll be in favor of reducing this wastage.  :)